src/test/resources/invalidCSV/input/instructions.csv

```diff
@@ -1,4 +1,4 @@
-ruimtereis01,Reis naar Centaur-planetoïde,Gewoon een stukje steen in de ruimte?,Captain,J.T.,,Kirk,,United Federation of Planets,Prof. Dr. Mr. Ir.,A.S.,,Terix,info:eu-repo/dai/nl/123456789X,Gallië,invalid-date,jan,dans,Een tweedaagse reis per ruimteschip naar een bijzondere planetoïde in de omgeving van Jupiter.,astronomie,,PALEOLB,abr:ABRperiode,here,degrees,83575.4,455271.2,,,,,ds1,,replaces,dummy,,Text with Qualifier,ISSUED,random test data,video/mpeg,NL,whatever,OPEN_ACCESS,1992-07-30,,path/to/audiofile/that/does/not/exist.mp3,,,other#domain,,other collection,,,,,user001
-ruimtereis01,Trip to Centaur asteroid,,,,,,,,,O.,van,Belix,,,,eko,knaw,A two day mission to boldly go where no man has gone before,ruimtevaart,,random text,,there,RD,,,1,2,3,4,,UNKNOWN_IDENTIFIER_TYPE,isRequiredBy,blabla,,,vAlId,,text/plain,encoding=UTF-8,,GROUP_ACCESS,invalid-date,D30000,,file title without file path,,,,,,,,,invalid-user
-ruimtereis01,,,,,,,,,,,,,,,,linda,,,IX,abr:ABRcomplex,,,,,,,,,,,,,,,http://x,30-07-1992,modiFied,,application/vnd.openxmlformats-officedocument.wordprocessingml.document,,,,,,,,,,,,,,,,
+DATASET,DC_TITLE,DCT_ALTERNATIVE,DCX_CREATOR_TITLES,DCX_CREATOR_INITIALS,DCX_CREATOR_INSERTIONS,DCX_CREATOR_SURNAME,DCX_CREATOR_DAI,DCX_CREATOR_ORGANIZATION,DCX_CONTRIBUTOR_TITLES,DCX_CONTRIBUTOR_INITIALS,DCX_CONTRIBUTOR_INSERTIONS,DCX_CONTRIBUTOR_SURNAME,DCX_CONTRIBUTOR_DAI,DCX_CONTRIBUTOR_ORGANIZATION,DDM_CREATED,DCT_RIGHTSHOLDER,DC_PUBLISHER,DC_DESCRIPTION,DC_SUBJECT,DC_SUBJECT_SCHEME,DCT_TEMPORAL,DCT_TEMPORAL_SCHEME,DCT_SPATIAL,DCX_SPATIAL_SCHEME,DCX_SPATIAL_X,DCX_SPATIAL_Y,DCX_SPATIAL_NORTH,DCX_SPATIAL_SOUTH,DCX_SPATIAL_EAST,DCX_SPATIAL_WEST,DC_IDENTIFIER,DC_IDENTIFIER_TYPE,DCX_RELATION_QUALIFIER,DCX_RELATION_TITLE,DCX_RELATION_LINK,DCT_DATE,DCT_DATE_QUALIFIER,DC_TYPE,DC_FORMAT,DC_LANGUAGE,DC_SOURCE,DDM_ACCESSRIGHTS,DDM_AVAILABLE,DDM_AUDIENCE,FILE_PATH,FILE_TITLE,FILE_ACCESSIBILITY,SF_DOMAIN,SF_USER,SF_COLLECTION,SF_PLAY_MODE,AV_FILE_PATH,AV_SUBTITLES,AV_SUBTITLES_LANGUAGE,DEPOSITOR_ID,BASE_REVISION
+ruimtereis01,Reis naar Centaur-planetoïde,Gewoon een stukje steen in de ruimte?,Captain,J.T.,,Kirk,,United Federation of Planets,Prof. Dr. Mr. Ir.,A.S.,,Terix,info:eu-repo/dai/nl/123456789X,Gallië,invalid-date,jan,dans,Een tweedaagse reis per ruimteschip naar een bijzondere planetoïde in de omgeving van Jupiter.,astronomie,,PALEOLB,abr:ABRperiode,here,degrees,83575.4,455271.2,,,,,ds1,,replaces,dummy,,Text with Qualifier,ISSUED,random test data,video/mpeg,NL,whatever,OPEN_ACCESS,1992-07-30,,path/to/audiofile/that/does/not/exist.mp3,,,other#domain,,other collection,,,,,user001,ab23-bdcg
```

Comment:
great to see you add an invalid UUID here! Hadn't even thought about this, but this way you can immediately test your new validation rule (`checkBaseRevisionConformsToUUID`)!

Reply:
after further reviewing, I would expect this error to show up in the test suite. However, I don't see it yet. I know you're not finished, but please don't forget to add the check.